frontend/src/metabase/qb/components/TimeseriesFilterWidget.jsx

```diff
@@ -103,16 +103,16 @@ export default class TimeseriesFilterWidget extends Component {
                 currentFilter
             ).join(" - ");
             if (currentFilter[0] === ">") {
-                currentDescription = "After " + currentDescription;
+                currentDescription = t`After`+ " " + currentDescription;
```

Comment:
I realize this was this way before you added the t tag but to make things a little cleaner I might convert these over to something like
```jsx
t`After ${currentDescription}`
```
just to keep things a bit more maintainable. Having to do `+ " "` to add an extra space could get a little fragile.